src/storybook/decorators.tsx

```diff
@@ -3,7 +3,7 @@ import { MemoryRouter } from 'react-router'
 
 // also exported from '@storybook/react' if you can deal with breaking changes in 6.1
 import { Frame } from 'components/common/Frame'
-import { StoryFnReactReturnType } from '@storybook/react/dist/client/preview/types'
+import { StoryFnReactReturnType } from '@storybook/react/dist/ts3.9/client/preview/types'
```

Comment:
Was this related to the PR?

Reply:
I also upgraded storybook, and the new version has a different path :shrug: